frontend/src/components/widgets/BaseWidget/WidgetLabel.tsx

```diff
@@ -16,23 +16,22 @@
  */
 
 import React from "react"
-import { StyledWidgetLabel } from "./styled-components"
+import {
+  StyledWidgetLabel,
+  LabelProps as LabelPropsT,
+} from "./styled-components"
 
-interface Props {
```

Comment:
From looking at other components, it seems like that styled-components should only contain Style-related props (in this case: `StyledWidgetLabelProps` with the `disabled` attribute). And the component file contains `Props` with all relevant attributes. For example:  https://github.com/streamlit/streamlit/blob/96f17464a13969ecdfe31043ab8e875718eb0d10/frontend/src/components/widgets/TextInput/styled-components.ts#L3

https://github.com/streamlit/streamlit/blob/96f17464a13969ecdfe31043ab8e875718eb0d10/frontend/src/components/widgets/TextInput/TextInput.tsx#L31

But I'm not sure if this is a general convention to follow.